projects/plugins/jetpack/extensions/blocks/podcast-player/edit.js

```diff
@@ -334,6 +344,7 @@ const PodcastPlayerEdit = ( {
 							max={ DEFAULT_MAX_ITEMS }
 							required
 							disabled={ !! selectedGuid }
+							help={ __( 'You can change number of items ranging from 1 to 100', 'jetpack' ) }
```

Comment:
```suggestion
							help={ __( 'You can change number of items ranging from 1 to 10', 'jetpack' ) }
```

Reply:
@kraftbj, after commiting your suggestion just now I noticed, it should be 100 only. Because as per the PR changes, max limit/value is going to be 100. Any user can change number of items from 1 to 100 and 10 is the default value. I believe I need to revert this commit.
What do you think?

Reply:
Taking a look at this previous [comment](https://github.com/Automattic/jetpack/issues/21291#issuecomment-944167521):

> Ideally, we'd have the range control UI update when one sets up the PHP filter.

It sounds to me like we'd want to pass down the value of the `jetpack_podcast_helper_list_quantity` filter to be used as the `DEFAULT_MAX_ITEMS`. Does that sound right @jeherve and if so, what would be the best way to pass that PHP filter value down to `edit.js`?

Reply:
> t sounds to me like we'd want to pass down the value of the jetpack_podcast_helper_list_quantity filter to be used as the DEFAULT_MAX_ITEMS

Yes, exactly.

> what would be the best way to pass that PHP filter value down to edit.js?

I would see 2 main ways to go about it:

- Add an endpoint to return that value, and call that endpoint from the js. Here is where we already define an endpoint for the podcast player: https://github.com/Automattic/jetpack/blob/5485111e6de594a9279bab856a63116b84762494/projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-podcast-player.php#L29-L32
- Pass the value via `Jetpack_Editor_Initial_State`.

I would personally vote for the former, but I'd be happy to be convinced differently. @samiff What's your take on this?

Reply:
Thanks Jeremy, I think adding an endpoint to return the filter value sounds good 👍  That value controls how many items are pulled from the feed, and also works nicely as a maximum for the block.

@amustaque97 would you like to give that approach a shot?

Reply:
:wae: @samiff, yes definitely I can try. I need some guidance on `add_filter` and `apply_filters`.

Jeremy mentioned  in the [comment](https://github.com/Automattic/jetpack/issues/21291#issuecomment-934258129) that adding following code will overcome limit constrains:

```php
add_filter(
	'jetpack_podcast_helper_list_quantity',
	function () {
		return 15;
	}
);
```

I tried in the local similar way at line number 187
<img width="896" alt="image" src="https://user-images.githubusercontent.com/21127788/140547541-74ace46a-d758-4b1b-b7ae-a9dbfb3ea1f6.png">

Now I'm calling a new endpoint as discussed in the above comments and still getting value as `10` in response because of apply_filter I believe.
<img width="1135" alt="image" src="https://user-images.githubusercontent.com/21127788/140547966-15ac53fc-60de-496f-9aa6-a9ecbb4a36c5.png">

During the actual API call to retrieve feeds it is returning 15 results. What am I doing wrong here?
<img width="1135" alt="image" src="https://user-images.githubusercontent.com/21127788/140548718-cd076f5c-13c0-45da-9c51-34c30d0bdb31.png">

Reply:
@samiff , I found issue  please ignore my previous comment. 😅